### ISSUE TITLE
PR oemof release 0.2.0

### ISSUE THREAD

**Issue by EdithaK** [oemof/organisation#25]
I would like to prepare the PR for the release 0.2.0. We should send news and mails about the new release to ensure high visibility. 
Therefore Miriam from RLI and I are writing a text from this fragments:
https://oemof.org/
http://oemof.readthedocs.io/en/latest/whats_new.html#v0-2-0
http://reiner-lemoine-institut.de/neue-oemof-version-jetzt-verfuegbar/
https://oemof.org/2017/03/27/the-oemof-springtime-release-v0-1-2/
https://oemof.org/2016/04/29/sunshine-release-v0-0-6/

I will post a first version within the next days here.
@all Please feel free to give suggestions for a nice text and/or graphics.

Suggested communication paths are:
* Openmod&Strommarkt mail distribution
* oemof news-Blog
* RLI-News-Blog
* News-Blog of the other institutions. 
@Non-RLI-developer: Can you give me a have a contact person or forward the text (when ready)?
* Probably we can also make a press release from it. 
@Non-RLI-developer: Do you agree that the RLI spreads the press release in the name of all?

**Comment by ckaldemeyer** [comment]
> I would like to prepare the PR for the release 0.2.0. We should send news and mails about the new release to ensure high visibility.
> Therefore Miriam from RLI and I are writing a text from this fragments:
> https://oemof.org/
> http://oemof.readthedocs.io/en/latest/whats_new.html#v0-2-0
> http://reiner-lemoine-institut.de/neue-oemof-version-jetzt-verfuegbar/
> https://oemof.org/2017/03/27/the-oemof-springtime-release-v0-1-2/
> https://oemof.org/2016/04/29/sunshine-release-v0-0-6/
> 
> I will post a first version within the next days here.
> @ALL Please feel free to give suggestions for a nice text and/or graphics.

Thanks!

> Probably we can also make a press release from it.
> @non-rli-developer: Do you agree that the RLI spreads the press release in the name of all?

Good idea. I would agree as long as HS/EUF/ZNES occur/are mentioned on the same level. Thanks!

**Comment by uvchik** [comment]
@EdithaK 

http://oemof.readthedocs.io/en/release-v0_2_0/whats_new.html#v0-2-0-january-12-2018

**Comment by uvchik** [comment]
> Good idea. I would agree as long as HS/EUF/ZNES occur/are mentioned on the same level. Thanks!

What about "ZNES helped to fixed some typos..." 😛

**Comment by uvchik** [comment]
By the way, the API changes are interesting for people who used version 0.1.x but nothing to promote foremost.

Highlights are the new features, the new components, the improved documentation, the lower barriers for new developers, more stable because of better tests and the new user forum to get help from other users and developers.

"If you update oemof from v0.1.x please note the API changes" or something like this.

**Comment by uvchik** [comment]
Any ideas for the name of the release?

**Comment by ckaldemeyer** [comment]
> Good idea. I would agree as long as HS/EUF/ZNES occur/are mentioned on the same level. Thanks!
> 
> What about "ZNES helped to fixed some typos..." :stuck_out_tongue:

I have again made some changes ;-)

> Any ideas for the name of the release?

Do we have any naming convention such as sweets in android, animals in ubuntu, ... ?!

**Comment by uvchik** [comment]
> Do we have any naming convention such as sweets in android, animals in ubuntu, ... ?!

😆 - you mean the stunning storage or the beautiful battery. No, there are no conventions and the name is not important. We could just say v0.2 release but that's a boring name for such an epic release 😛

**Comment by ckaldemeyer** [comment]
> Do we have any naming convention such as sweets in android, animals in ubuntu, ... ?!
> 
> :laughing: - you mean the stunning storage or the beautiful battery. No, there are no conventions and the name is not important. We could just say v0.2 release but that's a boring name for such an epic release :stuck_out_tongue:

Something like that would be funny :-)

**Comment by EdithaK** [comment]
The epic release ;)

**Comment by uvchik** [comment]
> The epic release ;)

This might be the `v1.0` ;-)

You could use the next coffee break to find a good name.... :-)

**Comment by uvchik** [comment]
Toddle Turbine ?

I will prepare the release tonight and will publish everything here and on PyPi tomorrow.

I think we should publish an oemof.org post on Monday (or would anybody prefer Friday).

All the other channels suggested by @EdithaK above could follow during the week.

@oemof/oemof-developer-group Additions, objections, approvals....?

**Comment by ckaldemeyer** [comment]
> Toddle Turbine ?

:+1: :smile: 



> I will prepare the release tonight and will publish everything here and on PyPi tomorrow.
> 
> I think we should publish an oemof.org post on Monday (or would anybody prefer Friday).
> 
> All the other channels suggested by @EdithaK above could follow during the week.
> 
> @oemof/oemof-developer-group Additions, objections, approvals....?

Nope. Thanks!

**Comment by uvchik** [comment]
I started to create a release note: Please (❗️) have a look (you have to be logged in).

https://github.com/oemof/oemof/releases

**Comment by c-moeller** [comment]
Thank you!
Can we edit directly? I found a typo.

**Comment by uvchik** [comment]
> Can we edit directly? I found a typo.

Yes

**Comment by uvchik** [comment]
It is released now but you still can fix typos by editing the message.

**Comment by uvchik** [comment]
It might be the base for the webpage post. Or we will link to this message from the webpage post. Proofreading this message is valuable - thanks.

**Comment by uvchik** [comment]
Who feels responsible for the different mailing lists:
 * openmod (BE)
 * strommarkt (DE)

Do you have news blogs in Flensburg (University?, ZNES?)

Please send the link to the release message (github/wordpress) to colleagues who might be interested.

**Comment by beritRLI** [comment]
I feel responsible - the version is released right? as I didn't find it on the news on the oemof site I was not sure. Shall I distribute the info now? Link to GitHub or also to the homepage?

**Comment by c-moeller** [comment]
Hi Berit, the news post will be published today. So maybe you wait until the news post is ready.

**Comment by ckaldemeyer** [comment]
> Who feels responsible for the different mailing lists:
> 
>     openmod (BE)
>     strommarkt (DE)
> 
> Do you have news blogs in Flensburg (University?, ZNES?)
> 
> Please send the link to the release message (github/wordpress) to colleagues who might be interested.

I would like to to that if that is o.k. for you @beritRLI ! Just because we put all the effort in it and this is one of the nice tasks :smile:

**Comment by ckaldemeyer** [comment]
> Who feels responsible for the different mailing lists:
> 
>     openmod (BE)
>     strommarkt (DE)
> 
>     Do you have news blogs in Flensburg (University?, ZNES?)
> 
>     Please send the link to the release message (github/wordpress) to colleagues who might be interested.
> 
> I would like to do that if that is o.k. for you @beritRLI ! Just because we put all the effort in it and this is one of the nice tasks :smile:

I have just talked to @uvchik and will do it this afternoon!

**Comment by uvchik** [comment]
> Please send the link to the release message (github/wordpress) to colleagues who might be interested.

But this is for everybody 😏

**Comment by beritRLI** [comment]
@ckaldemeyer 
sure!! I just wanted to help:)
You do Strommarkt too?

**Comment by ckaldemeyer** [comment]
> @ckaldemeyer
> sure!! I just wanted to help:)
> You do Strommarkt too?

Yep. I hope the text was o.k. for you! I tried to mention our institutions but also the aspect that it is a community project and everybody is welcome.

**Comment by EdithaK** [comment]
Thanks to Miriam, Cord and Uwe we now have this press release:
http://reiner-lemoine-institut.de/pressemitteilungen-oemof/
Cord already gave it to his PR-Office.
Thanks to Uwe we have an News-Block on the Website: https://oemof.org/2018/01/15/toddle-turbine-v0-2-0/
@beritRLI: You should have all the matirial for the mailing lists, now.

And than (I think) we did everything we wanted. Thanks!
